src/pages/ProviderProfile.jsx

```diff
@@ -0,0 +1,117 @@
+import {
+  Box,
+  Container,
+  Flex,
+  Image,
+  Text,
+  useTheme,
+  Button,
+  Skeleton,
+  HStack,
+} from "@chakra-ui/react";
+import React from "react";
+import { useParams } from "react-router-dom";
+import { ChevronRightIcon } from "@chakra-ui/icons";
+import { useProvider } from "../hooks";
+import ErrorPage from "./ErrorPage";
+import { useNavigate } from "react-router-dom";
+import ProviderInformation from "../components/provider-profile/ProviderInformation";
+
+const ProviderProfile = () => {
+  const { providerId } = useParams();
+  const { data, loading, error } = useProvider(providerId);
+  const theme = useTheme();
+  const navigate = useNavigate();
+
+  return (
+    <Box
+      bgColor={theme.colors.neutral[200]}
+      h="100%"
+      border={`1px solid ${theme.colors.neutral[400]}`}
+      py="24px"
+    >
+      {loading ? (
+        <Container maxW="800px">
+          <HStack spacing={2}>
+            <Skeleton height="500px" flex={1} />
+            <Skeleton height="500px" flex={2} />
+          </HStack>
+        </Container>
+      ) : error ? (
+        <ErrorPage error={error} />
+      ) : (
+        Object.keys(data).length > 0 && (
+          <Container maxW="800px">
+            <Flex>
+              <Text
+                fontSize="12px"
+                color={theme.colors.neutral[700]}
+                fontWeight={400}
+              >
+                Mental Wellness
+              </Text>
+              <Text fontSize="12px" mx="6px">
+                <ChevronRightIcon color={theme.colors.neutral[500]} />
+              </Text>
```

Comment:
This looks like a breadcrumb but these aren't actually links which are clickable making this a little confusing.